tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/SdkRelease/SdkReleaseTool.cs

```diff
@@ -0,0 +1,145 @@
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.ComponentModel;
+using System.Text.Json;
+using Azure.Sdk.Tools.Cli.Contract;
+using Azure.Sdk.Tools.Cli.Helpers;
+using Azure.Sdk.Tools.Cli.Models.Responses;
+using Azure.Sdk.Tools.Cli.Services;
+using Azure.Sdk.Tools.Cli.Tools.ReleaseReadiness;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.TeamFoundation.Common;
+using ModelContextProtocol.Server;
+
+namespace Azure.Sdk.Tools.Cli.Tools.SdkRelease
+{
+    [McpServerToolType, Description("This type contains the tools to release SDK package")]
+    public class SdkReleaseTool(IDevOpsService devopsService, ILogger<SdkReleaseTool> logger, ILogger<ReleaseReadinessTool> releaseReadinessLogger, IOutputService output) : MCPTool
+    {
+        private readonly string commandName = "sdk-release";
+        private readonly Option<string> packageNameOpt = new(["--package"], "Package name") { IsRequired = true };
+        private readonly Option<string> languageOpt = new(["--language"], "Language of the package") { IsRequired = true };
+        private readonly Option<string> branchOpt = new(["--branch"],() => "main",  "Branch to release the package from") { IsRequired = false };
+        public static readonly string[] ValidLanguages = { ".NET", "Python", "Java", "javaScript", "Go" };
+
+        public override Command GetCommand()
+        {
+            var command = new Command(commandName, "Run the release pipeline for the package") { packageNameOpt, languageOpt, branchOpt };
+            command.SetHandler(async ctx => { await HandleCommand(ctx, ctx.GetCancellationToken()); });
+            return command;
+        }
+
+        public async override Task HandleCommand(InvocationContext ctx, CancellationToken ct)
+        {
+            var packageName = ctx.ParseResult.GetValueForOption(packageNameOpt);
+            var language = ctx.ParseResult.GetValueForOption(languageOpt);
+            var branch = ctx.ParseResult.GetValueForOption(branchOpt);
+            var result = await ReleasePackageAsync(packageName, language, branch);
+            output.Output(result);
+        }
+
+        [McpServerTool(Name = "ReleasePackage"), Description("Releases the specified SDK package for a language. This includes checking if the package is ready for release and triggering the release pipeline. This tool calls CheckPackageReleaseReadiness")]
```

Comment:
Would it help to be more specific and say SDK?

```suggestion
        [McpServerTool(Name = "ReleaseSDKPackage"), Description("Releases the specified SDK package for a language. This includes checking if the package is ready for release and triggering the release pipeline. This tool calls CheckPackageReleaseReadiness")]
```